lib/utils.js

```diff
@@ -0,0 +1,89 @@
+/*!
+ * body-parser
+ * Copyright(c) 2014-2015 Douglas Christopher Wilson
```

Comment:
It's a new file, so it would be better to omit this or add that it should be express contributors.

Reply:
Should I remove the header completely?

Reply:
Probably this is the correct header only for this file, since we have to respect copyright rights. This happened on the website and the following copyright was added

```suggestion
 * Copyright(c) Express Contributors
```